index.html

```diff
@@ -0,0 +1,200 @@
+                        <div class="desk-card-content">
+                            <h5 class="desk-speaker-name">Mary Macaulay</h5>
+                            <p class="desk-speaker-qualification">Ph.D. in Agricultural Economics with expertise in food and agricultural policy.</p>
+                            <hr class="speaker-hr">
+                            <p class="desk-small-text">She conducted research on the impact of trade policies on agricultural production and exports</p>
+                        </div>
+                    </div>
+                    <div class="desk-card">
+                        <img class="desk-card-img" src="images/speaker-6.jpg" alt="speaker-picture">
+                        <div class="desk-card-content">
+                            <h5 class="desk-speaker-name">Kemi Anthony</h5>
+                            <p class="desk-speaker-qualification">Bachelor of Science in Agricultural Communications with experience in agricultural journalism and media relations.</p>
+                            <hr class="speaker-hr">
+                            <p class="desk-small-text">He collaborated with agricultural companies to develop effective marketing campaigns that increased sales and brand awareness.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+           
+            <div class="more-speakers">
+                <a class="show-more"  href="#"></a><button id="show-more" class="more-speakers-button">More <span><img class="angle-down" src="images/angle-down.png" alt="angle-down-icon"></span></button></a>
+            </div>
+        </div>
+    </section>
+    <section class="foot-section other-parts">
+        <div class="partners">
+            <h3 class="title-text move-foot">Partner<hr class="horizontal-line"></h3>
+            <div class="sponsor-logos">
+                <img class="sponsor-logo" src="images/spon-1.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-6.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-3.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-4.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-5.svg" alt="sponsor-logo">
+            </div>
+
+        </div>
+        <div class="main-footer">
+            <img class="about-footer-image" src="images/my-logo.png" alt="summit-logo">
+            <div class="footer-info">
+                <h4>2023 Agricultural Enthusiast Nigeria. All Rights Reserved</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Laudantium mollitia nisi, illo ratione voluptate ipsam perferendis cumque eaque inventore consequuntur aliquid at, aspernatur nesciunt corrupti minima expedita ea natus temporibus?</p>
+            </div>
+        </div>
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly use the `footer` tag and ensure it is in its own section

Reply:
- [ ] As pointed out by the previous reviewer kindly wrap the footer section in a `<footer></footer>` and not the `div` tag for semantic purposes